### PR TITLE
Fix some typos in unit descriptions

### PR DESCRIPTION
**Pull request by remram44**
Noticed some typos on http://zero-k.info/Static/UnitGuide

### CODE REVIEW

File: LuaUI/Configs/lang/units.en.json [hunk 1]
```diff
@@ -152,7 +152,7 @@
 	"turretgauss" : {
 		"name" : "Gauss",
 		"description" : "Popup gauss turret, regenerates when closed (Anti-Sub)",
-		"helptext" : "The Gauss is a compact, resilent turret with a medium-range gauss cannon. When popped down, it recieves a quarter of incoming damage as well as small amount of health regeneration. It can also attack underwater targets."
+		"helptext" : "The Gauss is a compact, resilient turret with a medium-range gauss cannon. When popped down, it receives a quarter of incoming damage as well as a small amount of health regeneration. It can also attack underwater targets."
```
Comment (Techhead0) [inline LuaUI/Configs/lang/units.en.json:155]: The damage reduction is no longer a quarter, and should say a third. The descriptions for the other armored units probably need checked as well

File: LuaUI/Configs/lang/units.en.json [hunk 2]
```diff
@@ -167,7 +167,7 @@
 	"cloakarty" : {
 		"name" : "Sling",
 		"description" : "Light Artillery Bot",
-		"helptext" : "The Sling has a long range plasma cannon that allows indirect fire over obstacles, and outranges static defenses up to heavy laser towers. Although effective versus mobile units, it should be guarded in order to prevent raiders and other fast units from closing range."
+		"helptext" : "The Sling has a long range plasma cannon that allows indirect fire over obstacles, and outranges static defenses up to heavy laser towers. Although effective versus mobile units, it should be guarded in order to prevent raiders and other fast units from close range."
```
Comment (Techhead0) [inline LuaUI/Configs/lang/units.en.json:170]: "closing" is correct

Comment (remram44) [reply LuaUI/Configs/lang/units.en.json]: oops!

File: LuaUI/Configs/lang/units.en.json (hunk 1 above)
Comment (remram44) [reply LuaUI/Configs/lang/units.en.json:155]: Also say a quarter: Crab, Solar Collector, Faraday, Lucifer, Desolator

File: LuaUI/Configs/lang/units.en.json [hunk 3]
```diff
@@ -692,7 +692,7 @@
 	"spideremp" : {
 		"name" : "Venom",
 		"description" : "Lightning Riot Spider",
-		"helptext" : "The Venom is an all-terrain unit designed to paralyze enemies so other units can easily destroy them. It moves particularly fast for a riot unit and in addition to paralysis it does a small amount of damage. Works well in tandem with the Recluse to keep enemies from closing range with the fragile skirmisher."
+		"helptext" : "The Venom is an all-terrain unit designed to paralyze enemies so other units can easily destroy them. It moves particularly fast for a riot unit and in addition to paralysis, it does a small amount of damage. Works well in tandem with the Recluse to keep enemies from close range with the fragile skirmisher."
```
Comment (Techhead0) [inline LuaUI/Configs/lang/units.en.json:695]: "Closing" is correct here too

**Comment by remram44** [comment]
I updated the armoured damage values: "quarter" -> "third", "4x" -> "3x"

**Comment by sprunk** [comment]
Thanks!

**Pull request merged**